### PR TITLE
Update default solc-compiler in truffle config

### PR DESCRIPTION
**Pull request by sukanyaparashar**
### ISSUE
`truffle-config.js` needs the latest solc version.

### SOLUTION
`truffle-config.js` is updated with the latest solc 0.8.13 version.

### CODE REVIEW

**Review by haltman-at — COMMENTED** [review]
Thanks for getting this!  One note: You might want to change the comment indentation back to how it was (this will require using `--no-verify` to get past `prettier`).  Or alternatively I guess you could put the spaces after the `//` so it won't complain?

(Similarly, `prettier` took out the comma, which I worry may cause inconvienence for users...)

File: packages/core/lib/commands/init/initSource/truffle-config.js [hunk 1]
```diff
@@ -100,17 +100,17 @@ module.exports = {
...
   // db: {
-    // enabled: false,
-    // host: "127.0.0.1",
-    // adapter: {
-    //   name: "sqlite",
-    //   settings: {
-    //     directory: ".db"
-    //   }
-    // }
+  // enabled: false,
```
Comment (haltman-at) [inline packages/core/lib/commands/init/initSource/truffle-config.js:107]: This is the comment spacing I was talking about.

File: packages/core/lib/commands/init/initSource/truffle-config.js [hunk 2]
```diff
@@ -91,7 +91,7 @@ module.exports = {
       //  evmVersion: "byzantium"
       // }
     }
-  },
+  }
```
Comment (haltman-at) [inline packages/core/lib/commands/init/initSource/truffle-config.js:94]: And this is the comma I was talking about.

**Review by haltman-at — COMMENTED** [review]
Sorry, I guess it might not have been entirely clear what spacing and commas I was referring to!

**Comment by sukanyaparashar** [comment]
> Sorry, I guess it might not have been entirely clear what spacing and commas I was referring to!

Sorry about the confusion @haltman-at . I have updated it.

**Review by haltman-at — APPROVED** [review]
Thanks for getting this Sukanya!

**Pull request merged**